### PR TITLE
document PII

### PR DESCRIPTION
**Pull request by malaffoon**
I checked java logging and ts console and it looks clean.

### CODE REVIEW

File: webapp/src/main/webapp/.gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /tmp
 /out-tsc
 /src/**/*.js
+!/src/scripts/iris/*.js
```
Comment (malaffoon) [inline webapp/src/main/webapp/.gitignore]: IntelliJ was complaining that these scripts "weren't part of the project" ... but they should be.

File: webapp/src/main/webapp/src/scripts/iris/util_xdm.js [hunk 2]
```diff
@@ -112,30 +112,27 @@ Util.XDM = (function ($) {
...
       // message request on the server from the client
-      console.log('MESSAGE REQUEST: ', data);
       var response = new Response(data);
       response.targetWindow = evt.source;
       response.targetOrigin = evt.origin === 'null' ? defaults.targetOrigin : evt.origin;
       response.send();
     }
     else if (data.type === 'response') {
       // message response from the server to the client
-      console.log('MESSAGE RESPONSE: ', data);
```
[the remark this answers is not in the document]
Comment (jtreuting) [reply webapp/src/main/webapp/src/scripts/iris/util_xdm.js]: yeah i can't imagine any reasons you can't delete those

[the remark this answers is not in the document]
Comment (wtritch) [reply webapp/src/main/webapp/src/scripts/iris/util_xdm.js]: yeah... I'm not sure if we added them while fighting with iris, or if the script came with those from the iris project... either way, they can't be required for functioning.

[the remark this answers is not in the document]
Comment (wtritch) [reply webapp/src/main/webapp/src/scripts/iris/util_xdm.js]: Unfortunately that won't get rid of all the console spam.  The i-framed external iris page is going to keep spamming the console...

**Pull request merged**